server/pulp/server/async/tasks.py

```diff
@@ -655,7 +655,7 @@ def cancel(task_id):
         consumer_id = tag_dict.get(tags.RESOURCE_CONSUMER_TYPE)
         agent_manager.cancel_request(consumer_id, task_id)
     else:
-        controller.revoke(task_id, terminate=True)
+        controller.revoke(task_id)
```

Comment:
I think with `terminate=True` it will cause a long-running task to be terminated immediately versus terminating after some time or when the task has completed. That is a behavior we need. Has a cancel been tested w/ this code to ensure that cancelling a running task cancels immediately and not later?